Finder.php

```diff
@@ -134,7 +134,19 @@ public function in($path)
         }
 
         foreach ($path as $p) {
-            $this->_paths[] = $p;
+            if (1 === preg_match('/[\*\?\[\]]/', $p)) {
```

Comment:
`preg_match` returns `false` if an error occured, which cannot happen in the `master` because tests will cover this regular expression and they will ensure it is correct ;-).
So it can return `0` or more if it finds more occurences. If it finds 2 occurences, this `if` condition will return `false`, so it's better to avoid the error case: `0 !== preg_match` is then better.

My thought :-).